datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -42,7 +42,8 @@ use std::fmt::{self, Debug, Display, Formatter};
 use std::hash::{Hash, Hasher};
 use std::sync::Arc;
 
-// backwards compatible
+// backwards compatibility
+pub use datafusion_common::display::{PlanType, StringifiedPlan, ToStringifiedPlan};
```

Comment:
For such things, I'm wondering as no deprecated message, it is possibly we can remove this kind of compatibility later?

Reply:
I tried to add a deprecated message (that could point people to use the new location) but it didn't seem to work for.

I think we can remove the `pub use` whenever we want, it will simply mean users of the crate will have to update their `pub use`s. 

I guess I am hoping that over time we can remove some of the old `pub use` but we don't really have a structured plan for doing so. 🤔

Reply:
Yea, for such re-exported things, seems it is impossible to add deprecated message. So the only way is just to remove this kind of old APIs after some time (e.g., few releases?).